tests/bearlib/aspects/ClassTest.py

```diff
@@ -56,3 +57,21 @@ class SubAspect:
         assert (str(exc.value) == "<class 'str'> is not an aspectclass or "
                 'an instance of an aspectclass')
         assert issubaspect(SubAspect('Python'), RootAspect)
+
+
+class AspectlistClassTest:
+
+    def test__init__(self):
+        with pytest.raises(TypeError) as exc:
+            aspectlist(['String'])
+        assert (str(exc.value) == "'String' is not an aspectclass or "
+                'an instance of an aspectclass')
```

Comment:
As mentioned in https://github.com/coala/coala/pull/3894#issuecomment-292197131 , I found out via https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions that you can simplify the exception message check to:

```python
with pytest.raises(TypeError) as exc:
    aspectlist(['String'])
exc.match("'String' is not an aspectclass or an instance of an aspectclass")
```

I will also simplify my exitsing ones :)